.spi.yml

```diff
@@ -1,5 +1,11 @@
 version: 1
 builder:
   configs:
+  - platform: ios
+    scheme: Parsing
+  configs:
+  - platform: tvos
+    scheme: Parsing
```

Comment:
We had to do the same in [swift-identified-collections](https://github.com/pointfreeco/swift-identified-collections/commit/680bf440178a78a627b1c2c64c0855f6523ad5b9#diff-5ebda4968fa267b5715182dbe59912825960a0c52655551a4f1c022b712ceaae) to get the platforms to pick things up. I think otherwise it tries to build the command line tools.